railties/lib/rails/generators.rb

```diff
@@ -116,6 +116,10 @@ def self.api_only!
         helper: false,
         template_engine: nil
       )
+
+      if ARGV.first == 'mailer'
+        options[:rails].merge!(template_engine: :erb)
```

Comment:
:+1: looking at the method more closely I think the comment above is outdated. It mentions that views are completely removed.